src/promdens/promdens.py

```diff
@@ -108,14 +111,23 @@ def __post_init__(self):
 
         if self.envelope_type == 'gauss':
             self.equation = "exp(-2*ln(2)*(t-t0)^2/fwhm^2)*cos((omega+chirp*t)*t)"
+            extent = 2.4 * self.fwhm
```

Comment:
Not sure about the name of this variable, maybe `temporal_extent`. Or something simpler?

Reply:
I would go with temporal_extent. It's long but it explains the meaning of the variable. I will change it in a next short pull request with some minor modifications.